### PR TITLE
Chat

### PR DESCRIPTION
**Pull request by xx-TOOL-xx**
Chat created. Added offers, changed contract connection
Screenshot from 2022-09-28 15-47-26

### CODE REVIEW

File: src/modules/job-post/entities/job-post.entity.ts [hunk 1]
```diff
@@ -14,12 +15,13 @@ import { Category } from 'src/modules/category/entities/category.entity';
 import { Skill } from 'src/modules/skill/entities/skill.entity';
 import { Users } from 'src/modules/entities/users.entity';
 import { LocalFile } from 'src/modules/entities/localFile.entity';
-import { Contracts } from 'src/modules/contracts/entities/contracts.entity';
+import { Offer } from '../../offer/entities/offer.entity';
```
Comment (jphinning) [inline src/modules/job-post/entities/job-post.entity.ts]: Absolute path

File: src/modules/offer/offer.controller.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  Post,
+  Request,
+  UseGuards,
+} from '@nestjs/common';
+import { JwtAuthGuard } from 'src/modules/auth/guards/jwt-auth.guard';
+import { searchParamDto } from 'src/modules/chat-room/dto/search-param.dto';
+import { AuthRequest } from 'src/utils/@types/AuthRequest';
+import { OfferService } from 'src/modules/offer/offer.service';
+import { Offer } from 'src/modules/offer/entities/offer.entity';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (jphinning) [inline src/modules/offer/offer.controller.ts:16]: Can you take either this comment or the unused variable out?

File: src/modules/job-post/entities/job-post.entity.ts (hunk 1 above)
Comment (xx-TOOL-xx) [reply src/modules/job-post/entities/job-post.entity.ts]: fixed

File: src/modules/offer/offer.controller.ts (hunk 2 above)
Comment (xx-TOOL-xx) [reply src/modules/offer/offer.controller.ts:16]: this is some eslint bug, but it shows that is unused variable, despite the fact that I'm using it in line 36
Screenshot from 2022-09-27 09-07-29

File: src/modules/offer/offer.controller.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    return this.offerService.getOneById(+id);
+  }
+
+  @UseGuards(JwtAuthGuard)
+  @Post()
+  create(@Body() createOfferDto: createOfferDto): Promise<Offer> {
+    return this.offerService.create(createOfferDto);
+  }
+
+  // @UseGuards(JwtAuthGuard)
```
Comment (lyudad) [inline src/modules/offer/offer.controller.ts]: hmm I think it should be uncomment, no?

**Pull request merged**